resources/data/ses_config-0.1.2.json

```diff
@@ -43,15 +43,20 @@
             "steps": [
                 {
                     "type": "Step",
-                    "title": "Choose paths",
+                    "title": "Import project",
                     "description": [
                         "Select **File > Open nRF Connect SDK Project**.",
                         "To import a project into SES, you must specify the following information:",
-                        "- **CMakeLists.txt** - the location of the ``CMakeLists.txt`` project file of the sample that you want to work with (for example, the full path to ``ncs/nrf/applications/asset_tracker/CMakeLists.txt``)",
-                        "- **Board Directory** - the location of the board description for which to build the project (for example, the full path to ``ncs/zephyr/boards/arm/nrf9160_pca10090``)",
-                        "- **Board Name** - the board name for which to build the project (for example, ``nrf9160_pca10090ns``)",
-                        "- **Build Directory** - the folder in which to run the build (for example, the full path to ``ncs/nrf/applications/asset_tracker/build``)",
-                        "- **Delete Existing CMakeCache.txt** - select this option to ensure that you are not building with an outdated build cache",
+                        "- **nRF Connect SDK Release** - Select the nRF Connect SDK version that you want to work with.",
+                        "- **nRF Connect Toolchain Version** - Select NONE and make sure that your SES environment is configured correctly.",
+                        "- **Projects** - Select the project that you want to work with.",
+                        "    Select any of the checkboxes to add samples and applications from the respective area to the drop-down list. To add custom projects to the drop-down list, click  and select the folder that contains the projects that you want to add.",
```

Comment:
Double space between "click" and "and" - not sure if that shows in the output.
The same in line 55.